app/models/host/managed.rb

```diff
@@ -295,6 +296,11 @@ def clear_data_on_build
     self.build_errors = nil
   end
 
+  def set_initiated_at
```

Comment:
It looked good for a moment, but this isn't right! :(   `initiated_at` gets set when a host is created, but it also gets reset when the host comes out of build mode.